rosa_getting_started/rosa_getting_started_iam/rosa-getting-started-workflow.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="rosa-understanding-the-deployment-workflow"]
+= Understanding the ROSA deployment workflow
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-understanding-the-deployment-workflow
+
+toc::[]
+
+Before you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you must complete the AWS prerequisites, verify that the required AWS service quotas are available, and set up your environment.
```

Comment:
Please double-check this assembly versus `rosa_architecture/rosa-sts-getting-started-workflow.adoc`. Some of the content doesn't appear to line up quite right, sort of like you've gotten a bit of the STS and non-STS workflow quickstarts crossed.